CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ development)_
 - New Message Component
 - Initial version of the Lookup component
 - Additional icons to the Icon component
+- New tokens based on wikimedia-ui-base v 0.18.0
```

Comment:
is this too generic, @itamargiv ? should I specify which tokens were added?

Reply:
Nah, I think this is good for now. Thank you :)